runtime/tools/python/setup.py

```diff
@@ -62,6 +63,24 @@
             f"{metallibdir}/{dylib}",
             f"{src_dir}/build/runtime/tools/python/ttrt/runtime",
         )
+        command = [
+            "patchelf",
+            "--set-rpath",
+            "$ORIGIN",
+            f"{src_dir}/build/runtime/tools/python/ttrt/runtime/{dylib}",
+        ]
+
+        try:
+            result = subprocess.run(
+                command,
+                check=True,
+                stdout=subprocess.PIPE,
+                stderr=subprocess.PIPE,
+                text=True,
+            )
+        except subprocess.CalledProcessError as e:
+            print(f"Command failed with return code {e.returncode}")
+
```

Comment:
can you remove all the changes from this file (apologies, I know this got caught in that merged revert!)

Reply:
Should be good now, thanks for pointing it out!